src/scripts/clipperUI/componentBase.ts

```diff
@@ -59,15 +59,15 @@ export abstract class ComponentBase<TState, TProps> {
 	}
 
 	/*
-	 * Helper which handles tabIndex, clicks, and keyboard navigation.
-	 *
-	 * Also hides the outline if they are using a mouse, but shows it if they are using the keyboard
-	 * (idea from http://www.paciellogroup.com/blog/2012/04/how-to-remove-css-outlines-in-an-accessible-manner/)
-	 *
-	 * Example use:
-	 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
-	 */
-	public enableInvoke(handleMethod: Function, tabIndex = 0, args?: any, idOverride: string = undefined) {
+   	* Helper which handles tabIndex, clicks, and keyboard navigation.
```

Comment:
This comment block is styled incorrectly (needs to use tabs everywhere, not spaces). Each asterisk between /* and */ should have an extra space liek before.